### PR TITLE
Feature/upload remotes

### PR DESCRIPTION
**Pull request by memsharded**
Changing UPLOAD and REMOTES to account for new bintray model.

Related to https://github.com/conan-io/conan/pull/1380

### CODE REVIEW

**Comment by lasote** [comment]
Add to README

File: conan/packager.py [hunk 1]
```diff
@@ -67,13 +67,22 @@ def __init__(self, args=None, username=None, channel=None, runner=None,
...
+        self.upload = upload or os.getenv("CONAN_UPLOAD", None)
         self.stable_branch_pattern = stable_branch_pattern or os.getenv("CONAN_STABLE_BRANCH_PATTERN", None)
         default_channel = channel or os.getenv("CONAN_CHANNEL", "testing")
         stable_channel = stable_channel or os.getenv("CONAN_STABLE_CHANNEL", "stable")
         self.channel = self._get_channel(default_channel, stable_channel)
 
+        if upload:
+            if upload in ("0", "None", "False"):
+                self.upload = None
+            elif upload == "1":
```
Comment (lasote) [inline conan/packager.py]: Will break ALL the builds

File: conan/packager.py [hunk 2]
```diff
@@ -67,13 +67,22 @@ def __init__(self, args=None, username=None, channel=None, runner=None,
         self.upload_retry = upload_retry or os.getenv("CONAN_UPLOAD_RETRY", 3)
         self.reference = reference or os.getenv("CONAN_REFERENCE", None)
         self.password = password or os.getenv("CONAN_PASSWORD", None)
-        self.remote = remote or os.getenv("CONAN_REMOTE", None)
```
Comment (lasote) [inline conan/packager.py]: Could break builds

File: conan/packager.py (hunk 1 above)
Comment (lasote) [inline conan/packager.py]: Confusing that the remote list cannot contain the upload remote. Conan will crash if added both?

Comment (memsharded) [inline conan/packager.py]: The builds will break anyway. It is necessary to specify the bintray remote to upload to, that is not handled by conan-package-tools now.

**Comment by lasote** [comment]
I'm adding it to my modified branch, I'll open a new PR.

**Pull request closed without merging**